examples/src/main/scala/org/apache/spark/examples/sql/streaming/StructuredNetworkWordCount.scala

```diff
@@ -68,6 +70,8 @@ object StructuredNetworkWordCount {
     val query = wordCounts.writeStream
       .outputMode("complete")
       .format("console")
+      .option("checkpointLocation",
+        "file:///" + System.getProperty("java.io.tmpdir") + "/spark_streaming_" + UUID.randomUUID())
```

Comment:
It's a bit nicer to use `Files.createTempDirectory` and call toURI if you need to rather than assemble it directly. It will deal with making the dir unique and all that.